packages/frontend/src/pages/UserPage.jsx

```diff
@@ -44,7 +45,20 @@ export default observer(() => {
             <h4>time</h4>
           </div>
           <div className='scroll'>
-            <UserEvent attester='s34setj09wkspq34sd' epoch='1' time='idk'/>
+            {/* {user.signUpsByUser[0] ? 
+              user.signUpsByUser.map(({ signup }) => (
+                <div className='event-card'>
+                  <Link to={`/attester/${signup.attesterId}`}>
+                    <p>0x<span>{signup.attesterId.slice(0, 2)}</span>...<span>{signup.attesterId.slice(-6, -1)}</span></p>
+                  </Link>
+                  <p>{signup.epoch}</p>
+                  <p>idk</p>
+                </div>
+              )) : null }
+            {user.signUpsByUser[0] ? 
+              null : 
+              'Loading...'
+            } */}
```

Comment:
I have changed the `UserEvent` component to accept the `signup` object instead of the values, but I'm first just trying here to iterate over the array `user.signUpsByUser`. I am continuing to try to understand how mobx is working here; I can see the values I'm trying access inside the proxy's target, but I don't understand how to get at them.
Even though I can get the length of this array as in the comment above, any attempt to access a value from one of the objects in the array returns
> react-dom.development.js:26918 Uncaught TypeError: Cannot read properties of undefined (reading 'attesterId')

Reply:
You have a typo on line 49. You're checking that `user.signUpsByUser[0]` is truthy, and then iterating using `user.signUpsByUser.map()`.

You should instead iterate using `user.signUpsByUser[0].map()`.

Reply:
Also on line 49 you're deconstructing a value `signup` from the array

```js
user.signUpsByUser.map(({ signup }) =>
```

Looking at the rest of the code I think this shouldn't be deconstructed, the signup object is not wrapped by anything. So the correct line might be:

```js
user.signUpsByUser[0].map((signup) =>
```

A valid deconstruction would be (accessing the variables on the signup object)


```js
user.signUpsByUser[0].map(({ attesterId, epoch }) =>
```